src/components/EmojiText.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Text from './Text';
+import * as EmojiUtils from '../libs/EmojiUtils';
+
+
+const propTypes = {
+    /** Text to display */
+    children: PropTypes.string,
+
+    /** Styles to be assigned to Container */
+    style: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    children: null,
+    style: {},
+};
+
+const EmojiText = props => (
```

Comment:
Let's leave a comment on why is this necessary. Why did we have to go through the EmojiText implementation?